templates/webmention-settings.php

```diff
@@ -137,6 +137,17 @@
 					</fieldset>
 				</td>
 			</tr>
+			<tr>
+				<th scope="row"><?php esc_html_e( 'Store Avatars', 'webmention' ); ?></th>
+				<td>
+					<fieldset>
+						<label for="webmention_avatar_store_enable">
+							<input type="checkbox" name="webmention_avatar_store_enable" id="webmention_avatar_store_enable" value="1" <?php echo checked( true, get_option( 'webmention_avatar_store_enable', 1 ) ); ?> />
```

Comment:
`checked` echos by default... we should change that for the other settings too...

```suggestion
							<input type="checkbox" name="webmention_avatar_store_enable" id="webmention_avatar_store_enable" value="1" <?php checked( true, get_option( 'webmention_avatar_store_enable', 1 ) ); ?> />
```